package-sign.yaml

```diff
@@ -55,7 +55,7 @@ tasks:
         $XDG_RUNTIME_DIR/gnupg/S.gpg-agent:/root/.gnupg/S.gpg-agent,
         $HOME/.gnupg/pubring.kbx:/root/.gnupg/pubring.kbx
       ]
-    target: package-sign-tarball WORKDIR=/projects/go-algorand
+    target: mule-sign-tar.gz WORKDIR=/projects/go-algorand
```

Comment:
Technically this is fine, stylistically the `.` in `tar.gz` kind of hurts my eyes...I'd prefer `targz`, but this is non-blocking.

Reply:
No, that's a variable.